code/modules/mob/living/living.dm

```diff
@@ -302,6 +302,11 @@
 
 // ++++ROCKDTBEN++++ MOB PROCS //END
 
+/mob/living/proc/getFeetAmount()
```

Comment:
There are a few other mob types that would be different than this too, like larva